eng/versioning/version_client.txt

```diff
@@ -19,8 +19,9 @@ com.azure:azure-cosmos-examples;4.0.1-beta.1;4.0.1-beta.1
 com.azure:azure-cosmos-benchmark;4.0.1-beta.1;4.0.1-beta.1
 com.azure:azure-data-appconfiguration;1.1.1;1.2.0-beta.1
 com.azure:azure-e2e;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-identity;1.0.4;1.1.0-beta.3
+com.azure:azure-identity;1.0.5;1.1.0-beta.4
 com.azure:azure-identity-perf;1.0.0-beta.1;1.0.0-beta.1
+>>>>>>> 3a255eb005748305722895bc50d3dd4ff5789bdb
```

Comment:
Looks like a bad merge here.

Reply:
Thanks! But you seems to be reviewing an outdated diff.